### PR TITLE
bindings: Handles undefined CPU and RAM Range max

### PR DESCRIPTION
**Pull request by HongjianZhang**
Before this commit, if user constructs CPU and RAM Range
with only one number, e.g. `new Range(2)`, the max of the
Range will be undefined, and inRange will return False.

This commit changes inRange function to treat undefined as
infinity.

### CODE REVIEW

File: blueprint/bindingsTest.js [hunk 1]
```diff
@@ -86,6 +86,12 @@ describe('Bindings', () => {
       checkInRange(2, 4, 4, true);
       checkInRange(2, 4, 2, true);
     });
+    it('undefined max', () => {
```
Comment (kayousterhout) [inline blueprint/bindingsTest.js]: Can you make this description a little more specific (so it's helpful to future folks updating the tests) -- something like 'undefined max results in a lower-bounded range'

File: blueprint/bindings.js [hunk 2]
```diff
@@ -1464,7 +1464,8 @@ class Range {
  * @returns {boolean} True if x is in the Range, and False otherwise.
  */
 Range.prototype.inRange = function inRange(x) {
-  return this.min <= x && (this.max === 0 || x <= this.max);
+  return this.min <= x &&
```
[the remark this answers is not in the document]
Comment (HongjianZhang) [reply blueprint/bindings.js]: This only happens when user does something like: `new Range()`, which isn't very likely, but I'll handle that just to be safe.

**Comment by HongjianZhang** [comment]
Fixed.

**Comment by kayousterhout** [comment]
Can you fix the commit name, consistent with the other PR?

**Comment by kklin** [comment]
What do we need to do for this to get merged? It's causing the [build to fail](https://travis-ci.org/kelda/mongo/builds/295458942) for some of our example blueprints, so I'd like to get it in.

**Comment by kayousterhout** [comment]
bump @ejj I think this is waiting on you

**Comment by kayousterhout** [comment]
@HongjianZhang can you update the commit message here?

**Pull request merged**